### DISCUSSION TITLE
can you handle failed messages?

### DISCUSSION THREAD

**Discussion by cilinjohnrey** [ibnux/Android-SMS-Gateway#16] [Ideas]
Can you resend failed messages in SMS app in android phone. I notice when i test the app i find some failed messages. I want to know if you can integrate it in this app

**Comment by ibnux** [comment]
I Will try it

I am currently added USSD to check balance, I will update with that too

**Reply by cilinjohnrey** [reply]
Ok sir, I will wait thank you very much

**Comment by cilinjohnrey** [comment]
How about the time interval in each sms can be send. I would like to implement every 1 sms will have 30 seconds to 60 seconds interval. If random much better.

**Comment by ibnux** [comment]
Retry failed sms 3 times with delay 10 second before retry. done. Maybe next time i will add configuration to change that value

for interval, you can create your own code, many SMS Gateway want send it fast. if you want to delay, delay with your own script

**Reply by cilinjohnrey** [reply]
Ok sir, I will wait for your changes
